pkg/auth/roles_authz.go

```diff
@@ -43,15 +41,12 @@ func NewAdminAuthZConfig() *AdminRoleAuthZConfig {
 func (c *AdminRoleAuthZConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.RolesConfigFile, "admin-authz-config-file", c.RolesConfigFile,
 		"Admin API authZ configuration file containing list of required role per API method")
-	fs.BoolVar(&c.Enabled, "enable-admin-authz", c.Enabled, "Enable admin API authZ via roles")
 }
 
 // ReadFiles will read and validate the contents of the configuration file.
 func (c *AdminRoleAuthZConfig) ReadFiles() error {
-	if c.Enabled {
-		if err := readRoleAuthZConfigFile(c.RolesConfigFile, &c.RolesConfig); err != nil {
-			return err
-		}
+	if err := readRoleAuthZConfigFile(c.RolesConfigFile, &c.RolesConfig); err != nil {
+		return err
 	}
 	return nil
```

Comment:
```suggestion
	return readRoleAuthZConfigFile(c.RolesConfigFile, &c.RolesConfig)
```
one small suggestion